edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Kubernetes/CombinedDockerModule.cs

```diff
@@ -68,14 +68,20 @@ public CombinedDockerModule(string version, ModuleStatus status, RestartPolicy r
         public virtual bool Equals(IModule<CombinedDockerConfig> other)
         {
             if (ReferenceEquals(null, other))
+            {
                 return false;
+            }
+
             if (ReferenceEquals(this, other))
+            {
                 return true;
+            }
+
             return string.Equals(this.Name, other.Name) &&
                 string.Equals(this.Version, other.Version) &&
                 string.Equals(this.Type, other.Type) &&
                 this.DesiredStatus == other.DesiredStatus &&
-                this.Config.Equals(other.Config) &&
+                this.Config.Equals(other.Config) && // TODO will compare only reference equality. Is this what we want?
```

Comment:
@darobs ?

Reply:
Definitely need something better here, or actually implement Equals for CombinedDockerConfig